travis-ci.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Don't fail silently when a step doesn't succeed
+set -e
+
+# if [ -z "$TRAVIS" ]; then
+#     echo "this script is intended to be run only on travis" >&2
+#     exit 1
+# fi
+
+make container deploy_kind
+
+echo "|---- Creating new Sonobuoy run/waiting for results..."
+./sonobuoy run --image-pull-policy=IfNotPresent -m Quick --wait & PID=$!
+
+# While process is running...
+while kill -0 $PID 2> /dev/null; do 
+    printf  "."
```

Comment:
You started off the conversation about this in slack and the idea of just outputting a stream of `.` seems the most simple answer. I opened #607 regarding this.

Reply:
I didn't want to wait for that so this is the easy fix. Showing output I think in the end game is best done with showing the total number of tests and which one is currently getting executed.

Reply:
Yea absolutely; I should have specified I didnt want that to be a blocker but just wanted to start the ticket since we had talked about it.